datastream-server-restli/src/test/java/com/linkedin/datastream/server/dms/TestDatastreamResources.java

```diff
@@ -559,11 +559,19 @@ public void testCreateDatastream() throws Exception {
     Set<String> missingFields = new HashSet<>();
 
     // happy path
-    Datastream fullDatastream = generateDatastream(1);
+    Datastream fullDatastream = generateDatastream(0);
     CreateResponse response = resource.create(fullDatastream);
     Assert.assertNull(response.getError());
     Assert.assertEquals(response.getStatus(), HttpStatus.S_201_CREATED);
 
+    Datastream whitespaceDatastream = generateDatastream(1);
```

Comment:
nit and feel free to ignore" add this as its own test. easy to spot perhaps?

Reply:
You're right. I'll add a comment to make it stand out instead because this test method contains all the datastream creation test cases.